### ISSUE TITLE
base16-chalk.dark Vim colors and too faded

### ISSUE THREAD

**Issue by OliverJAsh** [chriskempson/base16-iterm2#15]
I am using base16-chalk.dark in iTerm and opened Vim to see if the colours played well. Here is what I got for JavaScript file:

![image](https://cloud.githubusercontent.com/assets/921609/15858245/748db626-2cb7-11e6-8d01-bfb6022e6eb3.png)

What should I do to fix this? My understanding is base16-vim is only used for the GUI version. Do I need to use base16-iterm2 *and base16-vim*?

**Comment by aaron-williamson** [comment]
Yes, you're correct in guessing you need to use base16-vim if you want the colors to be like the GUI vim. For best results I would recommend using a terminal that supports truecolor (iTerm2 does support this, see [here](https://gist.github.com/XVilka/8346728)) as well as enabling `termguicolors` in vim by making sure you're running a recent enough vim and adding `set termguicolors` to your vimrc.

With my vim configured as such (though I also use a plugin for better javascript syntax highlighting, so it may differ a little) I get these results using only the base16-chalk vim colorscheme:
![image](https://cloud.githubusercontent.com/assets/9038848/20125409/96df9e7c-a5ea-11e6-8d90-031ebb637d12.png)
